src/main/resources/services/graphql/schema/query.js

```diff
@@ -113,7 +113,7 @@ module.exports = schemaGenerator.createObjectType({
             resolve: function (env) {
                 var query = env.args.query;
                 var itemIds = env.args.itemIds;
-                var count = env.args.count || Number.MAX_SAFE_INTEGER;
+                var count = env.args.count !== 0 ? (env.args.count || Number.MAX_SAFE_INTEGER) : 0;
                 var start = env.args.start || 0;
                 return useritems.list(null, query, itemIds, start, count);
```

Comment:
This code is duplicate of the one in `userItemsConnection.resolve`, except for nullable `types`. Should be extracted to a separate method which will be called from both places, f.ex.

```
getUserItems: function (args, types) {
  var query = args.query;
  var itemIds = args.itemIds;
  var count = args.count || 0;
  var start = args.start || 0;
  return useritems.list(types, query, itemIds, start, count);
}

userItemsConnection: {
  resolve: function (env) {
     return getUserItems(env.args.types, env.args);
  }
},

types: {
  resolve: function (env) {
     return getUserItems(null, env.args);
  }
},
```